site/_data/sidebar.yaml

```diff
@@ -28,6 +28,8 @@ groups:
     endpoint: metrics
   - name: Upgrade
     endpoint: upgrade
+  - name: BookKeeper http endpoint
```

Comment:
I would suggest changing "http endpoint" to "Admin REST API", which would be more accurate.

Reply:
thanks. will change it.